test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -372,6 +376,55 @@
                 [:webserver :ssl-crl-path]
                 "./dev-resources/config/jetty/ssl/crls/crls_bogus.pem")))))))
 
+(deftest crl-reloaded-without-server-restart-test
+  (let [tmp-dir (ks/temp-dir)
+        tmp-file (fs/file tmp-dir "mycrl.pem")
+        get-request #(http-get
+                      (str "https://localhost:8081" hello-path)
+                      default-options-for-https-client)]
+    (fs/copy "./dev-resources/config/jetty/ssl/crls/crls_none_revoked.pem"
+             tmp-file)
+    (with-app-with-config
+     app
+     [jetty9-service
+      hello-webservice
+      filesystem-watch-service/filesystem-watch-service]
+
+     (assoc-in
+      jetty-ssl-client-need-config
+      [:webserver :ssl-crl-path]
+      (str tmp-file))
+
+     (testing "request to jetty successful before cert revoked"
+       (let [response (get-request)]
+         (is (= (:status response) 200))
+         (is (= (:body response) hello-body))))
+
+     (testing "request fails after cert revoked"
+       ;; Sleep a bit to wait for the file watcher to be ready to poll/notify
+       ;; for change events on the CRL. Ideally wouldn't have to do this but
+       ;; seems like the initial event notification doesn't propagate if the
+       ;; file is changed too soon after initialization.
+       (Thread/sleep 1000)
+       (fs/copy "./dev-resources/config/jetty/ssl/crls/crls_localhost_revoked.pem"
+                tmp-file)
+       (loop [times 30]
+         (cond
+           (try
+             (ssl-exception-thrown? (get-request))
+             (catch IllegalStateException _
+               false))
+           (is true)
```

Comment:
@camlow325 I had it on my plate already so I don't mind pushing up a few changes today.